tests/functional/posting_test.php

```diff
@@ -72,6 +72,20 @@ public function test_html_entities()
 		$this->assertContains('&#128512;', $crawler->text());
 	}
 
+	public function test_quote()
```

Comment:
Please do make tests for other uses of the quote tag using the other attributes. There's no need to test all cases but do test in different cases (2 parameters, 2 parameters + 1 illegal parameter and 2 parameters without using the default attribute shortcut (`[quote author=""` instead of `[quote=`), for example).